### PR TITLE
Adding prefers-reduced-motion media feature

### PR DESCRIPTION
**Pull request by rachelandrew**
This is implemented in Firefox 63. I am about to add the page that it links to.

### CODE REVIEW

**Comment by Elchi3** [comment]
The corresponding Firefox bugs here are https://bugzilla.mozilla.org/show_bug.cgi?id=1479239 and https://bugzilla.mozilla.org/show_bug.cgi?id=1478505 (not supported on android yet).

**Comment by rachelandrew** [comment]
I was aware it isn't supported on Android, and didn't flag it as such. Is there something I need to change?

**Comment by Elchi3** [comment]
All good! :) 
In fact, you marked Firefox for Android as false and I was wondering if that's correct, so I researched these two bugs :)

**Comment by Elchi3** [comment]
Also correct: this landed for windows and macos in 63. See https://bugzilla.mozilla.org/show_bug.cgi?id=1475462. Caniuse is wrong about it.

File: css/at-rules/media.json [hunk 1]
```diff
@@ -1200,6 +1200,55 @@
...
+                "version_added": false
+              },
+              "firefox": {
+                "version_added": "63"
+              },
+              "firefox_android": {
+                "version_added": false
+              },
+              "ie": {
+                "version_added": null
```
Comment (Elchi3) [inline css/at-rules/media.json]: This should be false.

**Pull request merged**